### PR TITLE
Semitransparent background is now displayed when news are hovered or clicked

### PR DESCRIPTION
**Pull request by divyabaid16**
I did the issue #3337

### CODE REVIEW

File: frontend/src/app/components/UpdateList/index.scss [hunk 1]
```diff
@@ -118,10 +118,12 @@
 
     &:hover {
       border-color: $transparent-white-2;
+      background:rgba(255,255,255,0.05);
```
Comment (johngruen) [inline frontend/src/app/components/UpdateList/index.scss]: You can use the sass variable $transparent-white-05 here (https://github.com/mozilla/testpilot/blob/master/frontend/src/styles/_utils.scss)

File: frontend/src/app/components/UpdateList/index.scss [hunk 2]
```diff
@@ -118,10 +118,12 @@
...
     }
 
     &:active {
-      border-color: $transparent-white-5;
+      border-color: $transparent-white-4;
+      background:rgba(255,255,255,0.08);
```
Comment (johngruen) [inline frontend/src/app/components/UpdateList/index.scss]: for the background color, please add a new sass variable to this page (https://github.com/mozilla/testpilot/blob/master/frontend/src/styles/_utils.scss) then you can use the variable here!

**Review by johngruen — COMMENTED** [review]
@divyabaid16 thanks for your patch! I made a few comments for you to fix up.

**Comment by johngruen** [comment]
Great job!

File: frontend/src/app/components/UpdateList/index.scss [hunk 3]
```diff
@@ -118,10 +118,13 @@
...
     &:hover {
       border-color: $transparent-white-2;
+      background: $transparent-white-05;
     }
 
     &:active {
-      border-color: $transparent-white-5;
+      border-color: $transparent-white-4;
+      background: $transparent-white-08;
+
```
Comment (johngruen) [inline frontend/src/app/components/UpdateList/index.scss]: ooops! extra space in there...

**Comment by johngruen** [comment]
@divyabaid16 one more little fix for you then we can merge this!

**Comment by ghost** [comment]
Thanks!

**Pull request merged**